src/components/Expensicons/ClipboardIcon.js

```diff
@@ -0,0 +1,29 @@
+import React, {memo} from 'react';
+import PropTypes from 'prop-types';
+import ClipboardSvg from '../../../assets/images/clipboard.svg';
+import themeColors from '../../styles/themes/default';
+import variables from '../../styles/variables';
+
+const propTypes = {
+    isEnabled: PropTypes.bool.isRequired,
+    height: PropTypes.number,
+    width: PropTypes.number,
+};
+
+const defaultProps = {
+    height: variables.iconSizeNormal,
+    width: variables.iconSizeNormal,
+};
+
+const ClipboardIcon = props => (
+    <ClipboardSvg
+        height={props.height}
+        width={props.width}
+        fill={props.isEnabled ? themeColors.heading : themeColors.icon}
+    />
+);
```

Comment:
NAB, just an observation... we are creating a lot of components for all these SVG icons... and I'm starting to wonder when it's going to get old and we'll want to figure out a better way. A lot of them just have `height` and `width`. Seems like we should just have an `Icon` component that can take some different props and import an SVG when we want to use it then pass it to `Icon`.

Reply:
Yeah, that's exactly why I created [this PR](https://mail.google.com/mail/u/0/#label/Github/FMfcgxwKkRChtHclpqlMsjtNvGSnhxGV).

Reply:
Ah ok haha